Source/Fuse.Reactive.JavaScript/ValueMirror.uno

```diff
@@ -22,6 +22,7 @@ namespace Fuse.Reactive
 		readonly object _raw;
 		public object Raw { get { return _raw; } }
 		public virtual object ReflectedRaw { get { return _raw; } }
+		protected bool _isDisposed = false;
```

Comment:
I don't think this is the right place to introduce this. I think it's better to introduce these in ObjectMirror and ArrayMirror. And then we can maybe even name it a bit better...